modules/core/src/test/scala/org/scalasteward/core/nurture/UpdateInfoUrlFinderTest.scala

```diff
@@ -290,16 +273,10 @@ class UpdateInfoUrlFinderTest extends CatsEffectSuite with Http4sDsl[MockEff] {
   }
 
   test("possibleUpdateInfoUrls: on-prem Bitbucket Server") {
-    val forgeUrl = uri"https://bitbucket-server.on-prem.com"
-    val repoUrl = forgeUrl / "foo" / "bar"
-    val obtained = possibleUpdateInfoUrls(BitbucketServer, forgeUrl, repoUrl, v1, v2).map(_.url)
+    val repoUrl = uri"https://bitbucket-server.on-prem.com" / "foo" / "bar"
+    val obtained =
+      possibleUpdateInfoUrls(ForgeRepo(BitbucketServer, repoUrl), versionUpdate).map(_.url)
     val expected = repoUrl / "browse" / "ReleaseNotes.md"
     assert(clue(obtained).contains(expected))
   }
-
-  test("possibleUpdateInfoUrls: repoUrl is a home page") {
-    val repoUrl = uri"https://scalacenter.github.io/scalafix/"
-    val obtained = possibleUpdateInfoUrls(GitHub, uri"https://github.com", repoUrl, v1, v2)
-    assertEquals(obtained, List.empty)
-  }
```

Comment:
This assertion is deleted because it was testing that `possibleUpdateInfoUrls()` would return an empty list if called with a repo url that doesn't have an identifiable `ForgeType` - `possibleUpdateInfoUrls()` is now only ever called if a `ForgeType` has been identified, and that information is part of the required `ForgeRepo` parameter on the `possibleUpdateInfoUrls()` method.

Consequently this test no longer applies for the new method signature/parameter semantic! A similar assertion was deleted for `possibleVersionDiffs()` above.